btk_input.py

```diff
@@ -1,426 +0,0 @@
-import yaml
-import types
-import btk
-import os
-import numpy as np
-import imp
-import dill
-import subprocess
-
```

Comment:
Please write a few comment lines to explain what this file is doing.

Reply:
If I understand correctly, this handles the case when btk is ran with a configuration file?